controllers/bounce_processes_test.go

```diff
@@ -145,14 +141,18 @@ var _ = Describe("bounceProcesses", func() {
 		})
 
 		It("should kill the targeted processes", func() {
-			addresses := make([]string, 0, 1)
+			addresses := make(map[string]struct{}, 1)
 			for _, processGroupID := range []string{"storage-1"} {
 				processGroupAddresses := fdbv1beta2.FindProcessGroupByID(cluster.Status.ProcessGroups, processGroupID).Addresses
 				for _, address := range processGroupAddresses {
-					addresses = append(addresses, fmt.Sprintf("%s:4501", address))
+					addresses[fmt.Sprintf("%s:4501", address)] = struct{}{}
 				}
 			}
-			Expect(adminClient.KilledAddresses).To(ContainElements(addresses))
+			// NOTE: Previously, this didn't check for equality.
+			// It checked that KilledAddresses was a superset of addresses.
+			// This was changed in a refactoring, under the assumption the old
+			// behavior was a typo.
```

Comment:
For strings the `Equal` command will be fine in other places we used `HaveLen` and the `ContainElements` to check that all elements included. If I remember correctly we had some issued with the `Equal` statement and pointers.